documentation/modules/rn-2.2.adoc

```diff
@@ -98,3 +98,7 @@ If a Plan CR references storage, network, or VMs by name instead of by ID, the r
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.If a target VM is deleted, the VM migration status is still `Succeeded`
+
+If you delete a target `VirtualMachine` CR during the 'Convert image to kubevirt' step of the migration, the *Migration details* page of the web console displays the state of the step as `VirtualMachine CR not found`. However, the status of the VM migration is `Succeeded` in the `plan` CR file and in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2031529[*BZ#2031529*])
```

Comment:
Capitalize "plan" so that it is "Plan" CR. 

Named API objects follow camel caps conventions, like you did for "VirtualMachine" CR. 

See [OpenShift doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references).